backend/app/routers/auth_route.py

```diff
@@ -25,12 +25,18 @@
 
 @router.post('/send-verification', response_model=None)
 def send_verification(req: VerificationCodeCreate) -> None:
-    verification_status = send_verification_code(req.phone_number)
-    if verification_status == 'pending':
+    try:
+        verification_status = send_verification_code(req.phone_number)
+        if verification_status == 'failed':
+            raise HTTPException(
+                status_code=status.HTTP_400_BAD_REQUEST,
+                detail='Failed to send verification code',
+            )
+    except Exception as e:
         raise HTTPException(
             status_code=status.HTTP_400_BAD_REQUEST,
-            detail='Failed to send verification code',
-        )
+            detail=str(e),
+        ) from e
```

Comment:
Let's remove the broad except here. It doesn't add anything on top of normal FastAPI functionality and prevents us from seeing the error in the server logs.